src/Features/Core/Completion/CompletionItem.cs

```diff
@@ -16,6 +16,8 @@ internal class CompletionItem : IComparable<CompletionItem>
     {
         internal AsyncLazy<ImmutableArray<SymbolDisplayPart>> LazyDescription;
 
+        internal bool HasAsyncDescription { get; }
```

Comment:
This is a little unfortunate, but we've got plenty of other random bits already being set on CompletionItem. I'm currently working on formalizing the public Completion API. I'll work on figuring out a way to make it easier for hosts to handle custom data on CompletionItem so we don't have to keep adding random stuff to it (e.g. ShowsWarningIcon, ShouldFormatOnCommit, etc., etc.).